Dockerfile

```diff
@@ -17,12 +83,13 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
 
 # Set Timezone
 RUN echo "date.timezone = Europe/London" > /usr/local/etc/php/conf.d/timezone_set.ini
+# Set no memory limit (for PHP running as cli only).
+RUN echo "memory_limit = -1" > /usr/local/etc/php/conf.d/memory_limit.ini
```

Comment:
The comment above mentions this only applies to the CLI. How is that ensured? Is the option only honoured for CLI use?

Reply:
Updated the file to be `/usr/local/etc/php/php-cli.ini` this only gets picked up by php running in cli.